rel-eng/tito.props

```diff
@@ -0,0 +1,5 @@
+[buildconfig]
+builder = tito.builder.GemBuilder
+tagger = tito.tagger.VersionTagger
+changelog_do_not_remove_cherrypick = 0
+changelog_format = %s (%ae)
```

Comment:
Probably need the right koji tags here to:

[katello-nightly-rhel6]
disttag = .el6
blacklist =

[katello-nightly-rhel7]
disttag = .el7
blacklist =